L1Trigger/L1TMuonEndCap/python/simEmtfDigis_cfi.py

```diff
@@ -16,11 +16,20 @@
     CSCInput = cms.InputTag('simCscTriggerPrimitiveDigis','MPCSORTED'),
     RPCInput = cms.InputTag('simMuonRPCDigis'),
     GEMInput = cms.InputTag('simMuonGEMPadDigis'),
+    IRPCInput = cms.InputTag('simMuonRPCDigis'),
+    ME0Input = cms.InputTag('simMuonME0PadDigis'),
+    TTInput = cms.InputTag('TTStubsFromPhase2TrackerDigis','StubAccepted'),
 
     # Run with CSC, RPC, GEM
     CSCEnable = cms.bool(True),
     RPCEnable = cms.bool(True),
     GEMEnable = cms.bool(False),
+    IRPCEnable = cms.bool(False),
+    ME0Enable = cms.bool(False),
+    TTEnable = cms.bool(False),
+
+    # Era (options: 'Run2_2016', 'Run2_2017')
+    Era = cms.string('Run2_2017'),
```

Comment:
Are we allowed to have local era options?  How does this interface with the automatic configuration by FW version?  Again, @rekovic may have input ...